extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -20,6 +21,7 @@ define('ext.wikia.adEngine.video.videoSettings', [
 			state.autoPlay = isAutoPlay(params);
 			state.splitLayout = Boolean(params.splitLayoutVideoPosition);
 			state.moatTracking = Boolean(params.moatTracking);
+			state.withUiControls = Boolean(params.hasUiControls);
```

Comment:
I really don't like this typecasting. The double negation`!!` does the exact same thing, but is shorter. It seems to have better performance (https://www.sitepoint.com/javascript-double-negation-trick-trouble/) and from what I see is much more widely used in our repo.

What would need to happen if we stick to one, `!!` convention? @v3rth @fraszczakszymon @nandy-andy what do you think guys?